src/Symfony/Component/Config/Tests/Definition/NormalizationTest.php

```diff
@@ -182,8 +185,13 @@ public function testNonAssociativeArrayThrowsExceptionIfAttributeNotSet()
         $this->assertNormalized($this->getNumericKeysTestTree(), $denormalized, []);
     }
 
+    /**
+     * @group legacy
+     */
     public function testAssociativeArrayPreserveKeys()
```

Comment:
Do we have the equivalent (non-legacy) test where the only difference is that the node is configured as a map ?

Reply:
we did that implicitly, but I have now added an additional test case which checks for this explicitly